doc/control-tower.md

```diff
@@ -0,0 +1,160 @@
+# Path to AWS Control Tower
+
+[AWS Control Tower](https://aws.amazon.com/controltower/) provides an easy way to setup a Landing Zone in AWS: it will create a multi-account structure with proper set of permissions (using [AWS SSO](https://aws.amazon.com/single-sign-on/)) and guardrails (using [Service Control Policies](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) (SCPs) and [AWS Config](https://aws.amazon.com/config/)).
```

Comment:
I would emphasis that the multi accounts structure is mainly oriented around Security. This is where this bootstrap kit is different from CT since it tends to take care of the Software development life cycle (SDLC) accounts structures and that's where it makes sense at some point to mix them.